src/pages/one-page.js

```diff
@@ -12,7 +16,15 @@ var onePage = React.createClass({
     return (
       <div className={className}>
         <Header alt={this.context.intl.formatMessage({id: 'donate_to_mozilla'})}></Header>
-        <SingleForm/>
+        <SingleForm
```

Comment:
Why did this suddenly get lots o' props?

Reply:
because it was missing, and so it was just showing whatever magical defaults were baked in.